bot/modules/bartek/twitter_handler.py

```diff
@@ -0,0 +1,39 @@
+import aiohttp
+import logging
+import json
+import re
+
+from .text_handler import TextHandler
+
+
+class TwitterHandler:
+    def __init__(
+        self,
+        log: logging.Logger,
+        textHandler: TextHandler,
+        session: aiohttp.ClientSession,
+    ) -> None:
+        self.log: logging.Logger = log
+        self.textHandler: TextHandler = textHandler
+        self.session: aiohttp.ClientSession = session
+
+    async def isOffending(self, url: str) -> bool:
+        try:
+            # returns json embed data with some kind of description
+            async with self.session.get(
+                "https://publish.twitter.com/oembed?url=" + url
+            ) as resp:
+                twitter_embed_data_json: str = await resp.text()
+
+        except:
```

Comment:
A bare `except:` is bad practice. Please specify which exceptions we're expecting to catch.